en/snap/interactive-demo.md

```diff
@@ -1,46 +1,47 @@
-# Snap Interactive Integration Demo
+# Interactive Demonstration of Snap Integration
 <hr>
-Snap integration will be explained simply using interactive demo. Which you can interactively try the Snap integration demo below. Inspect the source code and see the live output.
 
-### Preparation
+Step 1 and Step 2 in [Snap Integration Overview](/en/snap/overview.md) is explained here using interactive demonstration. You can try the *Snap* integration demo given below. You can inspect the source code and see the real-time output.
+
+## Preparations
 <br>
 <div class="my-card">
 
 #### [Sign Up for Midtrans Account](/en/midtrans-account/overview.md)
-Sign up for an Account to get your Sandbox API keys ready to test integration.
+Sign up for a Midtrans Merchant Administration Portal (MAP) account, to get your API Keys for *Sandbox* environment and to test the integration.
 </div>
 
 <div class="my-card">
 
-#### [Retrieve API Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys)
-Retrieve Sandbox mode API keys that will be used for this guide.
+#### [Retrieving API Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys)
+Retrieve API Keys for *Sandbox* environment that will be used for this guide.
 </div>
 
-### Technology Stack
-
-The sample integration's:
-- **Backend** is **NodeJS** (hosted on CodeSandbox).
-- **Frontend** is plain **HTML & Javascript**.
-
-### Usage Explanation
+## Specifications
+The technical specifications of the demonstration integration are given below.
+- Backend: **NodeJS** (hosted on CodeSandbox)
+- Frontend: **HTML** and **JavaScript**
 
-You will see Snap Transaction Token as vizualization of API response on backend integration. Then you can click **"Proceed to Payment"** to test the frontend.
+## Usage Explanation
+You will see *Snap* Transaction Token as a visualization of API response on backend integration.
+  1. Click **Proceed to Payment** to test the frontend.
 
-?> Please wait until the window/iframe below is fully loaded. It may take some time as it try to build the backend.
+  > **Note**: Please wait until the window/iframe below is fully loaded. It may take some time while it tries to build the backend.
```

Comment:
As per the team discussion, the "Notes" inside the numbered steps use Blockquotes. Thus, this change is incorporated.